tutorials/auto_scheduler/tune_network_x86.py

```diff
@@ -205,7 +215,33 @@ def run_tuning():
         measure_callbacks=[auto_scheduler.RecordToFile(log_file)],
     )
 
-    tuner.tune(tune_option)
+    if use_sparse:
+        from tvm.topi.sparse.utils import (
+            sparse_conv2d_meet_condition_func,
+            sparse_conv2d_apply_func,
+            sparse_dense_meet_condition_func,
+            sparse_dense_apply_func,
+        )
+
+        search_policy = [
+            auto_scheduler.SketchPolicy(
+                task,
+                program_cost_model=auto_scheduler.XGBModel(),
+                init_search_callbacks=[
+                    auto_scheduler.PreloadCustomSketchRule(
+                        sparse_conv2d_meet_condition_func, sparse_conv2d_apply_func, "SparseConv2D"
+                    ),
+                    auto_scheduler.PreloadCustomSketchRule(
+                        sparse_dense_meet_condition_func, sparse_dense_apply_func, "SparseDense"
+                    ),
+                ],
+            )
+            for task in tasks
+        ]
```

Comment:
How about to pack these 4 condition func & apply func to a single entrance. For example:
```python
def sparse_sketch_rules() -> List[PreloadCustomSketchRule]
    ...

...
init_search_callbacks=sparse_sketch_rules(),
...
```

Reply:
Done.